package.json

```diff
@@ -28,6 +28,7 @@
     "test": "npm run test:unit",
     "test:unit": "vitest --environment jsdom",
     "type-check": "vue-tsc --noEmit -p tsconfig.vitest.json --composite false",
+    "circular-check": "tsc --showConfig -p ./tsconfig.app.json > temp.json && madge --ts-config ./temp.json --warning --circular --extensions ts,vue,js ./src",
```

Comment:
`temp.json` isn't deleted after completion, but I assume that's fine as it's only run in CI

Reply:
Indeed, I noticed that too, but didn't care much as this is just for CI!